### PR TITLE
CODEOWNERS: Give maintainer's code to github-sec team

### PR DESCRIPTION
**Pull request by pchaigno**
The new github-sec team will include more contributors than just the maintainers, to help scale reviews.

### CODE REVIEW

**Review by aditighag — APPROVED** [review]

**Comment by pchaigno** [comment]
Marking for backport to v1.10 to stop requiring reviews from maintainers on backport PRs.

**Pull request merged**
